assets/css/_circle_button.scss

```diff
@@ -0,0 +1,29 @@
+@use "./color/tokens_2024" as tokens;
+
+.c-circle-button {
+  width: 2rem;
+  height: 2rem;
+
+  background-color: tokens.$gray-50;
+  border-radius: 50%;
+
+  &:hover {
+    background-color: tokens.$gray-100;
```

Comment:
One additional nit - it looks like the hover state is overriding the active state, which I doubt is what design intended (it feels pretty weird to me). I can't quite wrap my mind around _why_ it's happening, since the nav menu looks like its CSS is set up pretty similarly, but it's not exhibiting this behavior.

That said, I don't think we should block merging for more than a very short time on this, so if we can get design feedback quickly on it, and/or if there's an obvious fix, then hurrah, but otherwise I'm good to merge and follow up with a fix (if indeed a fix is required)

Reply:
Ooooh I think I see the problem... sort of. It has to do with how CSS decides how to interpret conflicting rules.